fitbenchmarking/fitbenchmarking
# Adds system tests back

#### Description of Work

Fixes #632 


#### Testing Instructions

1. Check system tests are correctly set in the travis file

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
